lib/stats.rb

```diff
@@ -178,7 +178,18 @@ def teams_hash
   end
 
   def team_info
-
+    team_info_hash = Hash.new { |hash, key| hash[key] = [] }
+
+    @teams_data.each do |team|
+      team_info_hash[team[:team_id]] ={
```

Comment:
` = ` should always have only 1 space around it

Reply:
done